src/legacy/core_plugins/vis_type_metric/public/metric_vis_fn.ts

```diff
@@ -93,6 +93,7 @@ export const createMetricVisFn = (): ExpressionFunction<
     colorRange: {
       types: ['range'],
       multi: true,
+      default: '{range from=0 to=10000}',
```

Comment:
Had to explicitly setup default here, as if not provided, code fails in runtime in react component: 
https://github.com/elastic/kibana/blob/d86b6c72882af56b8fbc7cee02f5e2d5357274a1/src/legacy/core_plugins/vis_type_metric/public/components/metric_vis_controller.js#L97